### PR TITLE
Add support for Laravel 5.8

### PR DESCRIPTION
**Pull request by infernobass7**
Add support for Laravel 5.8 in composer file.

### CODE REVIEW

**Comment by monooso** [comment]
Hi Michael,

Thanks for the pull request.

As noted in the README, Unobserve _should_ work just fine in Laravel 5.8, but I haven't tested it. I'll need to set up a test environment for that before I can merge this.

Stephen

**Comment by infernobass7** [comment]
Hi Stephen,

Thank you for getting back to me. I am currently using a forked version on a project that is 5.8 and have not run into any issues.

Michael

File: composer.json [hunk 1]
```diff
@@ -26,8 +26,8 @@
     "prefer-stable": true,
     "require": {
         "php": ">=7.2.0",
-        "illuminate/contracts": "^6.0",
-        "illuminate/support": "^6.0"
+        "illuminate/contracts": "^6.0|5.8.*",
```
Comment (monooso) [inline composer.json]: Could you switch these around, to read `5.8.*|^6.0`. Same for `illuminate/support`.

**Comment by monooso** [comment]
Hi Michael,

Thanks for confirming that it works correctly with 5.8. If you can update the constraints as described above, I'll get this merged.

Cheers,
Stephen

**Pull request merged**